events/cloudwatch_logs.go

```diff
@@ -7,19 +7,19 @@ import (
 	"encoding/json"
 )
 
-// CloudwatchLogsEvent represents raw data from a cloudwatch logs event
```

Comment:
CloudWatch Events (now named Event Bridge) is distinct from CloudWatch Logs (still called CloudWatch!)

A find/replace of CloudWatch -> EventBridge isn't the way to go about this documentation refresh